scripts/release/build/rpm/package.sh

```diff
@@ -1,30 +1,34 @@
-#!/bin/bash -e
-
-# build_rpm.sh - Build a .deb package for one platform.
-#
-# Syntax:   build_rpm.sh <output directory>
-#
-# Examples: scripts/build_rpm.sh /tmp
+#!/usr/bin/env bash
 
 set -ex
 
-cd "$(dirname "$0")"/..
-export GOPATH=$HOME/subhome/go
-export REPO_DIR=${HOME}/subhome/go/src/github.com/algorand/go-algorand
-
 echo "Building RPM package"
 
+cd "$(dirname "$0")"/..
+
+REPO_DIR=${HOME}/subhome/go/src/github.com/algorand/go-algorand
+export REPO_DIR
```

Comment:
Consider consolidating assignment and export on a single line.

Reply:
This is a shellcheck suggestion that I've been following since we have brought that tool into our process.